index.js

```diff
@@ -68,7 +68,7 @@ function generateAuthToken() {
   //generate jwt auth
   const key = core.getInput('key')
 
-  const scopes = core.getInput('scopes')
+  const requiredScopes = ["ent_adobeio_sdk"]
```

Comment:
`ent_adobeio_sdk ` is a recommendation and not a requirement. Its possible to have workspace creds with a different API and still be able to deploy with JWT token. In such cases, where we hardcode `ent_adobeio_sdk ` and the API is actually not added to workspace will result in token generation error. 
Its better to leave the entire scope config in control of users.

Reply:
Ah I see, it's possible that the user's list of existing APIs might already have access to the scopes needed for deployment

I still feel like hard-coding will lead to less customer issues. Instead of the customer trying to figure out whether their existing APIs have access to the scopes needed to deploy, we just direct them to add this API to make sure. They won't know that their existing APIs might already have them covered, they will just see our setup steps in the README or the output message in the action logs directing them to add the I/O Management API and do it

If we go the route of still letting the customer configure their scopes, I feel like we should curate and provide a list of API's that have access to the scopes we need so they can check when their setting this up 

Maybe we can discuss in our next call

Reply:
Decision is to still support custom configuration of scopes, but make it optional. If the user doesn't provide a value for scopes, we fall back to the default of `["ent_adobeio_sdk"]` and direct them to add the I/O Management API